flink-streaming-java/src/main/java/org/apache/flink/streaming/api/environment/CheckpointConfig.java

```diff
@@ -382,7 +382,7 @@ public void setPreferCheckpointForRecovery(boolean preferCheckpointForRecovery)
 	}
 
 	/**
-	 * Enables unaligned checkpoints, which greatly reduce checkpointing times under backpressure.
+	 * Enables unaligned checkpoints, which greatly reduce checkpointing times under backpressure (experimental).
```

Comment:
As it's stable on our builds, maybe we could label it more production ready?

Reply:
Then just leave out experimental and just link to limitations?